ui/admin/app/templates/scopes/scope/host-catalogs/index.hbs

```diff
@@ -14,103 +14,129 @@
-            <B.Td>
-              <Hds::Copy::Snippet
-                @color='secondary'
-                @textToCopy={{B.data.id}}
-              />
-            </B.Td>
-          </B.Tr>
-        </:body>
-      </Hds::Table>
-
-    {{else}}
+  <page.body class='search-filtering'>
+    {{#if @model.hostCatalogsExist}}
+      <Hds::SegmentedGroup as |S|>
+        <S.TextInput
+          @value={{this.search}}
+          @type='search'
+          placeholder={{t 'actions.search'}}
+          aria-label={{t 'actions.search'}}
+          {{on 'input' this.handleSearchInput}}
+        />
+      </Hds::SegmentedGroup>
 
-      <Rose::Layout::Centered>
-        <Rose::Message
-          @title={{t 'resources.host-catalog.messages.welcome.title'}}
-          as |message|
+      {{#if @model.hostCatalogs}}
+        <Hds::Table
+          @model={{@model.hostCatalogs}}
+          @columns={{array
+            (hash label=(t 'form.name.label'))
+            (hash label=(t 'form.type.label'))
+            (hash label=(t 'form.id.label'))
+          }}
+          @valign='middle'
         >
-          <message.description>
-            {{#if (can 'list model' this.scope collection='host-catalogs')}}
-              {{! can list (at least):  show default welcome message}}
-              {{t 'resources.host-catalog.description'}}
-            {{else if
-              (can 'create model' this.scope collection='host-catalogs')
+          <:body as |B|>
+            <B.Tr>
+              <B.Td>
+                {{#if (can 'read host-catalog' B.data)}}
+                  <LinkTo
+                    @route='scopes.scope.host-catalogs.host-catalog'
+                    @model={{B.data.id}}
+                  >
+                    {{B.data.displayName}}
+                  </LinkTo>
+                {{else}}
+                  <Hds::Text::Body @tag='p'>
+                    {{B.data.displayName}}
+                  </Hds::Text::Body>
+                {{/if}}
+                <Hds::Text::Body
+                  @tag='p'
+                >{{B.data.description}}</Hds::Text::Body>
+              </B.Td>
+              <B.Td>
+                <HostCatalogTypeBadge @model={{B.data}} />
+              </B.Td>
+              <B.Td>
+                <Hds::Copy::Snippet
+                  @color='secondary'
+                  @textToCopy={{B.data.id}}
+                />
+              </B.Td>
+            </B.Tr>
+          </:body>
+        </Hds::Table>
+        <Rose::Pagination
+          @totalItems={{@model.totalItems}}
+          @currentPage={{this.page}}
+          @currentPageSize={{this.pageSize}}
+        />
+      {{else}}
+        <Hds::ApplicationState data-test-no-host-catalog-results as |A|>
+          <A.Header @title={{t 'titles.no-results-found'}} />
+          <A.Body
+            @text={{t
+              (if
+                this.search
+                'descriptions.no-search-results'
+                'descriptions.no-filter-results'
```

Comment:
heads up, this translation string was probs left over from filtering

Reply:
Woops great catch, I'll remove that